awx/main/migrations/0147_auto_20210610_2314.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 2.2.16 on 2021-06-10 23:14
```

Comment:
We should rename this migration, maybe something like `0147_rename_inv_sources.py` or something along those lines

Reply:
I have made the required changes , will be pushed in next commit.